src/test/java/MyFirstATest.java

```diff
@@ -0,0 +1,39 @@
+import com.codeborne.selenide.Selenide;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class MyFirstATest {
+    @Test
+    public void openPage() {
+        System.setProperty("webdriver.gecko.driver", "C:\\Users\\Acer\\Downloads\\geckodriver-v0.31.0-win64\\geckodriver.exe");
+        WebDriver driver = new FirefoxDriver();
+        driver.get("http://automationpractice.com/index.php");
+        driver.close();
+    }
+
+    @Test
+    public void openPage2() {
+        System.setProperty("webdriver.chrome.driver", "C:\\Users\\Acer\\Downloads\\chromedriver_win32\\chromedriver.exe");
+        WebDriver driver = new ChromeDriver();
+        driver.get("http://automationpractice.com/index.php");
+        driver.close();
+    }
+
+    @Test
+    public void openPage3() {
+        Selenide.open("http://automationpractice.com/index.php");
+    }
+
+    @Test
+    public void openPage4() {
+        System.setProperty("selenide.browser", "firefox");
```

Comment:
Please add screenshots of start Firefox browser by this test

Reply:
This is work?

Reply:
Yes this is work, I sent screenshots)

Reply:
![Screenshot_2](https://user-images.githubusercontent.com/93875949/164892541-454cdf2c-6fd2-4b00-b7bc-e02a7b0df03b.png)

Reply:
![Screenshot_1](https://user-images.githubusercontent.com/93875949/164892560-e6de9c0d-1178-4fb3-9f69-6759458349ed.png)

Reply:
![Screenshot_3](https://user-images.githubusercontent.com/93875949/164892575-4268a4af-0f91-462e-86a1-5d059665fe0b.png)